### PR TITLE
[Monorepo] ESlint Nullish Operator

### PR DESCRIPTION
**Pull request by JacobMGEvans**
Add the nullish coalescing operator preferred rule to ESLint

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -49,6 +49,10 @@ module.exports = {
 				"no-mixed-spaces-and-tabs": ["error", "smart-tabs"],
 				"no-shadow": "error",
 				"require-yield": "off",
+				"@typescript-eslint/prefer-nullish-coalescing": [
```
Comment (petebacondarwin) [inline .eslintrc.js:52]: I am not really keen on adding "warning" rules. We should either error or not say anything.

**Review by petebacondarwin — COMMENTED** [review]
If we care about this then we should error, right?

**Comment by IgorMinar** [comment]
+1 for erroring. Use //ignore comments to exempt and justify when we need to disable the check for particular code

**Comment by JacobMGEvans** [comment]
Going to rework this another time.

**Pull request closed without merging**